src/components/Menu/MenuItem.tsx

```diff
@@ -138,8 +137,9 @@ export function MenuItem({
       </li>
     );
   } else if (
-    (currentPlatform && pageNode?.platforms?.includes(currentPlatform)) ||
-    !pageNode.platforms
+    !pageNode.hideFromNav &&
```

Comment:
I don't think we actually want the Contribute page to be nested under [platform] so I added this value we can set in the meta object for a page `hideFromNav` which will allow us to put it in our directory tree so it can still get breadcrumb support but hide from the main nav.